src/index.js

```diff
@@ -311,7 +311,15 @@ class WarmUp {
     const warmUpFunction = `"use strict";
 
 /** Generated by Serverless WarmUp Plugin at ${new Date().toISOString()} */
-const aws = require("aws-sdk");
+let aws;
+try {
+  const AWSXRay = require("aws-xray-sdk-core");
+  aws = AWSXRay.captureAWS(require("aws-sdk"));
+}
+catch {
+  console.log("X-Ray SDK not found, falling back to basic AWS SDK.");
```

Comment:
I would not notify if X-Ray is not there but the other way around.
I'd notify if X-Ray is used.

Reply:
++ That makes sense.